### PR TITLE
finished project for homework week2 Jalal Al Attar

### PR DESCRIPTION
**Pull request by Jalalalattar**
Review

### CODE REVIEW

File: Week2/homework/js-exercises/3-theRecipeCard.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        'salt and pepper',
+        'spaghetti',
+        'eggs',
+    ]
+};
+console.log(pasta);
+
+
+for(let propertyOutSeperately in pasta){
+    console.log(`${propertyOutSeperately} is ${pasta[propertyOutSeperately]}`);
```
Comment (marcianoviereck) [inline Week2/homework/js-exercises/3-theRecipeCard.js:18]: nicenice!

File: Week2/homework/js-exercises/4-readingList.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+    console.log(myBooks[i].title ,' by ', myBooks[i].author);
+
+    if (myBooks[i].alreadyRead === true){
+        console.log('I already read ' , myBooks[i].title); // 4
+    }else{
+        console.log('I still need to read ' , myBooks[i].title); // 5
+    }
+
+};
```
Comment (marcianoviereck) [inline Week2/homework/js-exercises/4-readingList.js:53]: haha, really nice!

File: Week2/homework/js-exercises/gradeCalculator.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    case (score >= 0 && score <=59):
+      output = 'F';
+      break;
+    default:
+      output = 'INVALID SCORE'
+    }
+    return `You got a ${output} ${score}%`;
+  };
+  let c = Math.floor((Math.random() * 100) + 1);
+  console.log(convertScoreToGrade(c));
```
Comment (marcianoviereck) [inline Week2/homework/js-exercises/gradeCalculator.js:52]: very nice!

**Review by marcianoviereck — APPROVED** [review]
Well done, the homework is looking really clean and good!

File: Week2/homework/js-exercises/gradeCalculator.js (hunk 3 above)
Comment (Jalalalattar) [reply Week2/homework/js-exercises/gradeCalculator.js:52]: this week without `var` :)

**Pull request closed without merging**